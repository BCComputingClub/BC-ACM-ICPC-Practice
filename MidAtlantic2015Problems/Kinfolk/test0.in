1 5 M
1 11 F
0 8 F
5 7 M
0 32767 F
-1 -1 M
