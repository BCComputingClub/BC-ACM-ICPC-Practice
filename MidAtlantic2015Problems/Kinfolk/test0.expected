nephew
grandniece
great-grandchild
1st cousin once removed
kin
